facade/pool.go

```diff
@@ -200,7 +200,6 @@ func (f *Facade) RestoreResourcePools(ctx datastore.Context, pools []pool.Resour
 	var alog audit.Logger
 	for _, pool := range pools {
 		alog = f.auditLogger.Message(ctx, "Adding ResourcePool").Action(audit.Add).Entity(&pool)
-		pool.DatabaseVersion = 0
```

Comment:
Have you tried a back up and restore? I'm not sure if having the primaryTerm and seqNo here will hurt.

Reply:
without primaryTerm and seqNo, there was an issue with restoring